pmlc/ast/builder.cc

```diff
@@ -752,6 +768,106 @@ struct ProgramBuilder {
         .result();
   }
 
+  Value handleLoop(ExprNodeLoop *node) {
+    // take lowBound, upperBound, step from operands.
+    auto indexs = node->indexs;
```

Comment:
Typo: index.   Correct word is indexes. We should use indices. However, I think here we should use three variables lower-bound, upper_bound and step, instead of a vector.